cli/flox/src/commands/services/status.rs

```diff
@@ -68,10 +68,28 @@ struct ProcessStateDisplay {
     name: String,
     status: String,
     pid: u64,
+    exit_code: Option<i32>,
     #[serde(skip_serializing)]
     is_running: bool,
 }
 
+impl From<ProcessState> for ProcessStateDisplay {
+    fn from(proc: ProcessState) -> Self {
+        ProcessStateDisplay {
+            name: proc.name,
+            status: proc.status,
+            pid: proc.pid,
+            // process-compose uses 0 to indicate no exit code.
+            exit_code: if proc.is_running || proc.exit_code == -1 {
```

Comment:
What does `-1` mean?

Is it unrelated to the comment about `0`?

Reply:
had to update the comment.

its 0 for _running_ processed and -1 for stopped processes.
but 0 does not make sense for running processes so rather then passing on the exit codes we set None here in both cases